apps/test/netsim/NetSimTable.js

```diff
@@ -0,0 +1,231 @@
+var testUtils = require('../util/testUtils');
+var assert = testUtils.assert;
+var NetSimTable = testUtils.requireWithGlobalsCheckBuildFolder('netsim/NetSimTable');
+
+var assertEqual = function (left, right) {
+  assert(left === right, left + '===' + right);
+};
+
+/**
+ * Storage table API placeholder for testing, always hits callbacks immediately
+ * so tests can be written imperatively.
+ * @returns {Object}
+ */
+var fauxStorageTable = function () {
```

Comment:
I'd like to see this generalized so that any app that needs to mock the API can re-use it, but I'm ok with leaving this here until a second consumer needs to refactor it out into something shared.